jme3-core/src/plugins/java/com/jme3/texture/plugins/TGALoader.java

```diff
@@ -463,29 +502,139 @@ public static Image load(InputStream in, boolean flip) throws IOException {
 
 
         in.close();
+
+        return resizeImage(width, height, pixelDepth, format, rawData);
+    }
+
+    /**
+     * Change this image if need.
+     *
+     * @param width current width of image
+     * @param height current height of image
+     * @param pixelDepth pixel depth of image
+     * @param format format of image.
+     * @param rawData data of image.
+     * @return result image.
+     */
+    protected Image resizeImage(int width, int height, final int pixelDepth, final Format format, byte[] rawData) {
```

Comment:
Also, I'll mention that personally I don't think resizing images at run-time is a good idea, because it slows down loading for people who want lower texture quality. Further, resizing is not possible to do for compressed formats like DXT or ETC. A better option might be to use a mipmap level offset when specifying the texture in OGL, so that it ignores mipmap level 0 for example, effectively halving the texture resolution.